app/webpacker/packs/school_search.js

```diff
@@ -50,7 +50,72 @@ function getCheckedSchools() {
   return [];
 }
 
-const editSchoolFormEl = document.querySelector("form.edit_school");
+function getSubmitButton() {
+  return document.querySelector(".partnerships-submit");
+}
+
+function getClearButton() {
+  return document.querySelector("#clear-button");
```

Comment:
We should discuss JS element selection (but it's a wider discussion than this PR). Will start a thread on slack.